pkg/rotator/rotator.go

```diff
@@ -537,43 +537,18 @@ func ValidCert(caCert, cert, key []byte, dnsName string, at time.Time) (bool, er
 	return true, nil
 }
 
-// controller code for making sure the CA cert on the
-// webhooks don't get clobbered
-
-var _ handler.Mapper = &crdMapper{}
-
-type crdMapper struct {
-	secretKey types.NamespacedName
-	crdNames  []string
-}
-
-func (m *crdMapper) Map(object handler.MapObject) []reconcile.Request {
-	if object.Meta.GetNamespace() != "" {
-		return nil
-	}
-	for _, crdName := range m.crdNames {
-		if object.Meta.GetName() == crdName {
-			return []reconcile.Request{{NamespacedName: m.secretKey}}
+func reconcileSecretAndWebhookMapFunc(webhook WebhookInfo, r *ReconcileWH) func(object client.Object) []reconcile.Request {
+	return func(object client.Object) []reconcile.Request {
+		whKey := types.NamespacedName{Name: webhook.Name}
+		object.GetNamespace()
```

Comment:
why is `object.GetNamespace()` called here?

Reply:
ah, good call, this line looks like it doesn't do anything and should probably be removed.